README.md

```diff
@@ -1,4 +1,4 @@
-# AWS SDK for Go
+# AWS SDK for Go (with modified SQS API that uses JSON wire protocol)
```

Comment:
Is the intention to have this PR merged upstream? If so, consider removing this note.

Reply:
No, I won't be able to merge upstream due to permission. This PR is intended for customers to try out the private Beta prior to JSON GA.